### ISSUE TITLE
Live preview doesn't open a new window of chrome if chrome is already open.

### ISSUE THREAD

**Issue by Jackbennett** [adobe/brackets#11057]
When a user, staff or domain admin clicks to run the live preview _and_ already has chrome open. Brackets always opens a new tab in chrome. This doesn't work as it won't have remote debugging enabled. 

If I allow brackets to try to close and re-open chrome it again just opens a new tab. 

The user must already have chrome closed first and allow brackets to open it. 

My environment:
* Windows 7 SP1 x64
* I've deployed brackets on my domain with a computer install MSI GPO.
* Multi user x64bit chrome msi installed to `...Program files(x86)...`
* Brackets 1.3
* `File > Experimental "Live preview"` is disabled. This is a fresh install it was never enabled.

I couldn't find any string in the brackets source on github that contains `chrome.exe` or `--remote-debugging` as I was wondering if I might need to add `--new-window`

**Comment by redmunds** [comment]
This is by design. Brackets automatically uses the `--remote-debugging` flag (see https://github.com/adobe/brackets-shell repo). Is this causing any problems, or just unexpected?

**Comment by Jackbennett** [comment]
If chrome is already open. Brackets won't launch a new window. It makes a new tab and hangs loading the `file:///C:/Program%20Files%20(x86)/Brackets/www/LiveDevelopment/launch.html` because the first chrome wasn't launched by brackets with `--remote-debugging`

How do I get brackets to force opening a new window? This works like you'd expect on my laptop but on the domain the open chrome captures the new tab.

**Comment by Jackbennett** [comment]
[Is this the line?](https://github.com/adobe/brackets-shell/blob/b31e5cb11f24957788876ec7819b8f6dd843b632/appshell/appshell_extensions_win.cpp#L344)

append `--new-window` on that string and I guess brackets would be opening a new window all the time for live preview. I assume this would always make a new window when clicking preview.

Perhaps if the boolean `enableRemoteDebugging` was instead some sort of reference to the opened process with debugging on it would re-connect to the open window. Else open with `--new-window`.

Could that way reconnect there after a user opened devtools?

**Comment by js0511** [comment]
NAVER - http://www.naver.com/
--------------------------------------------

oiup2006@naver.com 님께 보내신 메일 <Re: [brackets] Live preview doesn't open a new window of chrome if chrome is already open. (#11057)> 이 다음과 같은 이유로 전송 실패했습니다.

--------------------------------------------

받는 사람이 회원님의 메일을 수신차단 하였습니다. 


--------------------------------------------

**Comment by redmunds** [comment]
This feature request would have to be done as a preference, since I think most people prefer to have a new tab opened so there are less windows to manage.

**Comment by Jackbennett** [comment]
I agree but there's also a subtle bug in here. Brackets will just try to open new chrome tabs in a window that it can never connect to. The relaunch "chrome button" that it offers after timing out just keeps opening new tabs. 

I see in the network console brackets makes a request to 127.0.0.1:9222 before chrome even starts to open. Maybe if brackets sees this return `connection refused` it knows to use `--new-window` Or just use the new windows param for the "relaunch chrome" button on the failed connection popup.

Does brackets get back a message when chrome has opened to avoid the first GET always failing anyway when previewing?

**Comment by redmunds** [comment]
> Brackets will just try to open new chrome tabs in a window that it can never connect to. The relaunch "chrome button" that it offers after timing out just keeps opening new tabs. 

There is some other connection problem going on. Start by reading the [Live Preview section of the How To Use Brackets doc](https://github.com/adobe/brackets/wiki/How-to-Use-Brackets#live-preview) and the [Live Preview Isn't Working section of the Brackets Troubleshooting guide](https://github.com/adobe/brackets/wiki/Troubleshooting) to make sure this isn't a known problem.

**Comment by mgla56** [comment]
Live Preview не работает с php что делать
